.github/workflows/checks.yml

```diff
@@ -226,12 +225,32 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v4
 
+    - name: Install Movement CLI
+      run: |
+        sudo apt-get update 
+        sudo apt-get install -y build-essential
+        sudo apt-get install -y binutils
+        sudo apt-get install -y lld
+        sudo apt-get install -y libudev-dev
+        sudo apt-get install -y libdw-dev
+        which ld
+        which lld
+        which gcc
+        which cc
+        echo $PATH
+        export GIT_CLONE_PROTECTION_ACTIVE=false
+        git clone https://github.com/movementlabsxyz/aptos-core/ 
+        cd aptos-core
+        cargo build -p movement
+        sudo cp target/debug/movement /usr/local/bin/
```

Comment:
There's a space right after movement and is this essentially aptos cli but renamed? Are we not colliding with our own cli that will bring in tooling for bridging?

Reply:
Yes, it's Movement CLI. I don't think it would collide with the bridge CLI in the sense of them having different purposes. From what I can tell, the bridge CLI is for bridge-related commands like doing a swap. 

Movement CLI can spin up a Movement network, specifically, is the reason why I chose use it for this.

But we'll want to use the bridge `MovementClient` for transacting with the bridge modules. 

Now that you mention it, it's not the `MovementClient` publishing the package (which is fine, as it's closer to the purpose of Movement CLI rather than the bridge CLI) so I need to change the name of that test. This is basically my way of doing something like a simplified Anvil on Movement, with Movement CLI. 

I looked at [this](https://github.com/aptos-labs/aptos-core/blob/538c7aef902c963c3bd9da8dc5c8b07bac840924/aptos-move/e2e-move-tests/src/harness.rs) from Aptos but since `MovementClient` is supposed to be signing transactions it seemed like a lot of that functionality would collide with testing the client and utilities that are in the bridge.